alvr/client_openxr/src/extra_extensions/passthrough_fb.rs

```diff
@@ -54,6 +54,8 @@ impl PassthroughFB {
             layer_handle,
         };
 
+        unsafe { super::xr_res((ext_fns.passthrough_start)(handle))? };
+
```

Comment:
By spec, this call is not needed when using the flag `IS_RUNNING_AT_CREATION`. Let's add a comment this is a workaround for PFD.

Reply:
ah true, I guess I should bug them then